style.css

```
@@ -0,0 +1,71 @@
+header {
+	display: flex;
+	justify-content: flex-end;
+}
+a {
+    color: #222222;
+    font-size: 20px;
+    font-family: Arial, Helvetica, sans-serif;
+    text-decoration: none;
+}
+.menu {
+	position: fixed;
+	top: 0;
+	left: 0;
+	width: 100%;
+	height: 100%;
+	padding: 15px;
+    background: #FFEFBA;
+	transform: translateY(-100%);
+	transition: transform 0.5s; 
+}
+
+.menu.active {
+	transform: translateY(0);
+}
+
+.menu li {
+	list-style-type: none;
+}
+
+
+.menu-btn {
+	width: 30px;
+	height: 30px;
+	position: relative;
+	z-index:2;
+	overflow: hidden;
+}
+
+.menu-btn span {
+	width: 30px;
+	height: 2px;
+	position: absolute;
+	top: 50%;
+	left: 50%;
+	transform: translate(-50%, -50%);
+	background-color: #222222;
```

Краще використовувати скорочений запис
```suggestion
	background-color: #222;
```